core/templates/pages/about-page/about-page.component.html

```diff
@@ -1,106 +1,112 @@
 <div class="oppia-static-header">
-  <h1 translate="I18N_ABOUT_PAGE_HEADING"></h1>
+  <h1 [innerHTML]="'I18N_ABOUT_PAGE_HEADING' | translate"></h1>
```

Comment:
Why is this changed as `[innerHTML] = | translate`?

Reply:
I did it for the tags in the content.